eva/storage/storage_engine.py

```diff
@@ -12,10 +12,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from eva.catalog.catalog_type import TableType
+from eva.catalog.models.df_metadata import DataFrameMetadata
 from eva.configuration.configuration_manager import ConfigurationManager
+from eva.storage.abstract_storage_engine import AbstractStorageEngine
 from eva.utils.generic_utils import str_to_class
 
-StorageEngine = str_to_class(ConfigurationManager().get_value("storage", "engine"))()
-VideoStorageEngine = str_to_class(
-    ConfigurationManager().get_value("storage", "video_engine")
-)()
+
+class StorageEngine:
+    storages = {
+        TableType.STRUCTURAL_DATA: str_to_class(
+            ConfigurationManager().get_value("storage", "engine")
```

Comment:
Nit: Would it make sense to change the "engine" here to `structured_data_engine`? I'm assuming that the other modalities will also be added as `audio_engine` etc.

Reply:
Yes, makes sense. Will do it